libdnf.spec

```diff
@@ -35,6 +35,7 @@ BuildRequires:  gcc
 BuildRequires:  libsolv-devel >= %{libsolv_version}
 BuildRequires:  pkgconfig(librepo)
 BuildRequires:  pkgconfig(check)
+BuildRequires:  redhat-rpm-config
```

Comment:
The issue will not show up in mock builds (since `@buildsys-build` installs `redhat-rpm-config`), so just have `redhat-rpm-config` installed on your system.